treeherder/model/derived/jobs.py

```diff
@@ -347,7 +349,8 @@ def cancel_job(self, requester, job):
                            project_specific_id=job['id'],
                            state='pending').update(
                                state='completed',
-                               result='usercancel')
+                               result='usercancel',
+                               last_modified=datetime.now())
```

Comment:
I thought the custom `.save()` was meant to do this?
https://github.com/mozilla/treeherder/blob/573de7c7ce19aad94aba846d357b65d21a90cc07/treeherder/model/models.py#L748